jailer/src/lib.rs

```diff
@@ -33,45 +33,45 @@ pub enum Error {
     CgroupInheritFromParent(PathBuf, String),
     CgroupLineNotFound(String, String),
     CgroupLineNotUnique(String, String),
-    ChangeFileOwner(sys_util::Error, &'static str),
-    ChdirNewRoot(sys_util::Error),
-    CloseNetNsFd(sys_util::Error),
-    CloseDevNullFd(sys_util::Error),
+    ChangeFileOwner(io::Error, &'static str),
```

Comment:
Also update the implementation of `Display` to use to_string instead of format with debug.

Reply:
My bad. For the errors in Vmm there were formatted with Debug ("{:?}"), but it looks like these ones are already formatted with Display ("{}").